lib/tentacat/repositories/merges.ex

```diff
@@ -3,10 +3,12 @@ defmodule Tentacat.Repositories.Merges do
   alias Tentacat.Client
 
   @doc """
-  Merge a branch into another
+  Merge a branch into another.
 
   You can use both branch names and/or a commit SHA1.
+
   When the commit message is empty, Github will apply the default commit message.
+
```

Comment:
why are we adding new lines for each sentence?
I haven't gone through the doc, but wouldn't this introduce too many white space throughout the page? 😐

Reply:
> why are we adding new lines for each sentence?
> I haven't gone through the doc, but wouldn't this introduce too many white space throughout the page?

First sentence as short summary of the function. I've combined the remaining sentences as a paragraph.